raiden/storage/migrations/v19_to_v20.py

```diff
@@ -73,14 +108,13 @@ def _add_onchain_locksroot_to_channel_settled_state_changes(
     for state_changes_batch in batch_query:
         updated_state_changes = list()
         for state_change in state_changes_batch:
-            data = json.loads(state_change.data)
+            state_change_data = json.loads(state_change.data)
```

Comment:
here you just renamed this ... nothing to do with the bugfix, or?

Reply:
Yea... seemed like i am decoding the JSON twice, so i removed one of them.

Reply:
all right.